backend/z_index.js

```diff
@@ -36,6 +37,26 @@ app.get('/absences', async (req, res) => {
   const absences = await getAbsenceForms();
   res.json(absences);
 })
+app.get('/meetingHistory', async (req,res) => {
+  const meetingHist = await getMeetingHistory();
+  res.json(meetingHist);
```

Comment:
nit: make sure you either do res.json or return res.json

FYI:
return res.json ... : stops execution of the function
https://stackoverflow.com/questions/22902099/nodejs-is-it-a-bad-practise-to-return-res-json